### PR TITLE
Make record open in generated code in sending-a-message-to-a-service tutorials

### PR DESCRIPTION
**Pull request by dulajdilshan**
## Purpose
$title

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/integration-tutorials/sending-a-message-to-a-service.md [hunk 1]
```diff
@@ -142,13 +142,13 @@ You have successfully developed the required service.
 import ballerina/http;
 import ballerina/log;
 
-type Doctor record {|
+type Doctor record {
```
Comment (dulajdilshan) [inline swan-lake/integration-tutorials/sending-a-message-to-a-service.md:145]: Related comment: https://github.com/ballerina-platform/ballerina-dev-website/pull/8133/files#r1380952869

File: swan-lake/integration-tutorials/sending-a-message-to-a-service.md [hunk 2]
```diff
@@ -58,18 +58,21 @@ Follow the instructions given in this section to develop the service.
...
         decimal fee;
-    |};
+    };
     ```
 
     > **Note:**
     > While it is possible to work with the JSON payload directly, using record types offers several advantages including enhanced type safety, data validation, and better tooling experience (e.g., completion).
 
+    > **Note:**
+    > Opted for a [closed record](https://ballerina.io/learn/by-example/records/) in the source code to emphasize the type-safety and maintainability.
```
Comment (MaryamZi) [inline swan-lake/integration-tutorials/sending-a-message-to-a-service.md:74]:
```suggestion
    > When the fields of the JSON objects are expected to be exactly those specified in the sample payload, the generated records can be updated to be [closed records](https://ballerina.io/learn/by-example/controlling-openness/), which would indicate that no other fields are allowed or expected.
```

Comment (dulajdilshan) [reply swan-lake/integration-tutorials/sending-a-message-to-a-service.md:74]: I feel like, The link you have provided mainly tells that making a record an open record to allow other fields. But shouldn't we emphasize more on making the records closed?

Comment (MaryamZi) [reply swan-lake/integration-tutorials/sending-a-message-to-a-service.md:74]: No, right? The first sentence `Use record {| ... |} to describe a record type that allows exclusively what is specified in the body.` and the first example show exactly closed records.

Comment (dulajdilshan) [reply swan-lake/integration-tutorials/sending-a-message-to-a-service.md:74]: As a second thought, It tells both open and closed records. Let's put this. +1

File: swan-lake/integration-tutorials/sending-a-message-to-a-service.md [hunk 3]
```diff
@@ -58,18 +58,21 @@ Follow the instructions given in this section to develop the service.
...
         decimal fee;
-    |};
+    };
     ```
 
     > **Note:**
     > While it is possible to work with the JSON payload directly, using record types offers several advantages including enhanced type safety, data validation, and better tooling experience (e.g., completion).
 
+    > **Note:**
+    > When the JSON object fields are intended to precisely match the sample payload, the generated records can be updated to be [closed records](https://ballerina.io/learn/by-example/controlling-openness/), which would indicate that no other fields are allowed or expected.
```
Comment (MaryamZi) [inline swan-lake/integration-tutorials/sending-a-message-to-a-service.md:74]: I don't think " the JSON object fields are intended" reads well. We expect the fields of the expected payload to match the sample payload.

Any issue with the suggestion in https://github.com/ballerina-platform/ballerina-dev-website/pull/8180#discussion_r1381150777?

Comment (dulajdilshan) [reply swan-lake/integration-tutorials/sending-a-message-to-a-service.md:74]: Ah no. Just wanted to improve

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**